src/main/kotlin/app/FactCodes.kt

```diff
@@ -1,10 +1,20 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
 package app
 
 object FactCodes {
     val COMMITS_DAY_WEEK = 1
     val COMMITS_DAY_TIME = 2
+    val COMMITS_LINE_NUM_AVG = 8
+    val COMMITS_NUM = 9
+    // Used for number of commits per number of lines in a commit histogram.
+    // Key should be number of lines. Value number of commits.
+    val COMMITS_NUM_PER_LINE_NUM = 12
```

Comment:
I find the name a bit confusing. Perhaps replacing 'per' to 'to' would make nicer? For example, COMMIT_COUNT_TO_LINE_COUNT. Also COMMITS -> COMMIT.